lib/resin-lint.ts

```diff
@@ -5,6 +5,25 @@ import * as glob from 'glob';
 import * as optimist from 'optimist';
 import * as path from 'path';
 import * as tslint from 'tslint';
+import { promisify } from 'util';
+
+// TODO: Switch to using fs.promises when dropping node 8 support
+const realpathAsync = promisify(fs.realpath);
+const readFileAsync = promisify(fs.readFile);
+const accessAsync = promisify(fs.access);
+const statAsync = promisify(fs.stat);
+const writeFileAsync = promisify(fs.writeFile);
+
+const existsAsync = (filename: string) => {
+	try {
+		accessAsync(filename);
+		return true;
+	} catch {
+		return false;
+	}
+};
+
+const globAsync = promisify(glob);
```

Comment:
If you have tested it and it works then it's fine... The question is whether `glob` is a function, given the import style:
```
import * as glob from 'glob';
```
I don't know if it would make any difference to use [Typescript's import = require()](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) syntax:
```
import glob = require('glob');
```

Reply:
Yeah it's a function (typescript would also catch if it wasn't) and the output between `import *` and `= require` is the same (`const glob = require("glob");`). I tend to prefer the `import *` style wherever possible though as it feels more typescript-y to me